api.bs

```diff
@@ -645,23 +680,25 @@ The arguments to <a method for=PrivateAttribution>measureConversion()</a> are as
   <dt><dfn>filterData</dfn></dt>
   <dd>Only [=impressions=] having a filterData value matching this value will be eligible to match this [=conversion=].</dd>
   <dt><dfn>impressionSites</dfn></dt>
-  <dd>A list of impression sites. Only [=impressions=] recorded where the top-level site is on this list are eligible to match this [=conversion=].</dd>
+  <dd>A list of impression sites. Only [=impressions=] recorded where the [=impression site=] is on this list are eligible to match this [=conversion=].</dd>
   <dt><dfn>intermediarySites</dfn></dt>
   <dd>
     A list of sites which called the <a method for=PrivateAttribution>saveImpression()</a> API.
-    Only [=impressions=] recorded by scripts originating from one of the intermediary sites
+    Only [=impressions=] recorded by scripts originating from one of the [=intermediary sites=]
     are eligible to match this [=conversion=].
   </dd>
 </dl>
 
 
 ### Operation ### {#measure-conversion-api-operation}
 
-1. Collect the implicit API inputs
-    1. The current timestamp
-    2. The conversion site domain
-    3. The iframe site domain
-1. Validate the page-supplied API inputs
+1.  Collect the implicit API inputs from the [=relevant settings object=]:
+    1.  The timestamp is set to the [=current high resolution time=].
+    2.  The [=conversion site=] is set to the [=site=] from the [=top-level origin=].
+    3.  The [=intermediary site=] is set to the [=site=] from [=origin=]
+        if that is not [=same site=] with the [=top-level origin=].
+        Otherwise, the value is `undefined`.
```

Comment:
We can reference https://html.spec.whatwg.org/multipage/browsers.html#obtain-a-site for these.